src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -235,6 +252,7 @@ class AccessionStore(
                         .set(FAMILY_NAME, accession.family)
                         .set(FIELD_NOTES, accession.fieldNotes)
                         .set(FOUNDER_ID, accession.founderId)
+                        .set(IS_MANUAL_STATE, if (accession.isManualState) true else null)
```

Comment:
Storing this as true/null rather than true/false mostly so it doesn't force me to add `isManualState = false` to all the `AccessionsRow` objects in various test classes.